src/hyp_map.rs

```diff
@@ -577,7 +590,7 @@ impl HypMap {
             .next()
             .unwrap();
```

Comment:
.expect() was added as an improvement to the unwrap() that was all over the initialization code.

I think in this respect this unwrap, and others if exist, should be equally handled.

In general, I'd like error happening during initialization to be clearly marked as fatal. Either in comments or names.

Reply:
I agree, it would be a lot of work but it would be nice to know that there is no scope for panic.

Reply:
In this case would be a `ok_or` ? :-)

Reply:
Yes `.ok_or` is a good solution for places like this.

Reply:
And hasn't been changed why?